common/religion/religions/daradorism.txt

```diff
@@ -57,7 +57,44 @@
 	}
 
 	custom_faith_icons = {
-		custom_faith_1 custom_faith_2 custom_faith_3 custom_faith_4 custom_faith_5 custom_faith_6 custom_faith_7 custom_faith_8 custom_faith_9 custom_faith_10 dualism_custom_1 zoroastrian_custom_1 zoroastrian_custom_2 buddhism_custom_1 buddhism_custom_2 buddhism_custom_3 buddhism_custom_4 taoism_custom_1 yazidi_custom_1 sunni_custom_2 sunni_custom_3 sunni_custom_4 ibadi_custom muhakkima_1 muhakkima_2 muhakkima_4 muhakkima_5 muhakkima_6 judaism_custom_1
+		## Warcraft
+		# Fauna
+		wcmw_custom_generic_owl_1
+		## Vanilla
+		# Flora
+		# Leaf
+		baltic_custom
+		# Vine
+		custom_faith_3
+		# Tree
+		custom_faith_6 waaqism custom_faith_fp1_yggdrasil
+		# Flower
+		custom_faith_10
+		# Fauna
+		# Elephant
+		custom_faith_5
+		# Horse
+		custom_faith_7
+		# Cat
+		custom_faith_9
```

Comment:
Don't think there is a lot of cats in Northrend.

Reply:
These religion are present in both Northrend and EK